docker-compose.yml

```diff
@@ -1,12 +1,21 @@
 version: '3'
 services:
+
     web:
-        build: ./
-        container_name: svgtranslate_web
+        build:
+            dockerfile: ./.docker/web/Dockerfile
+            context: .
         ports:
-          - ${TOOLFORGE_DOCKER_PORT}:80
+            - ${TOOLFORGE_DOCKER_PORT}:80
         volumes:
-          - ./.docker/lighttpd.conf:/etc/lighttpd/lighttpd.conf:cached
-          - ./:/var/www:cached
+            - ./:/var/www:cached
         environment:
-          TOOLFORGE_DOCKER_PORT: ${TOOLFORGE_DOCKER_PORT}
+            TOOLFORGE_DOCKER_PORT: ${TOOLFORGE_DOCKER_PORT}
+
+    assets:
+        build:
+            dockerfile: ./.docker/assets/Dockerfile
```

Comment:
I think you can remove this `Dockerfile` (and this `build` section) and put:
```
image: node:11.5
working_dir: /svgtranslate
```
that will save you from having to build the container. :)